01-ruby-fundamentals/exercises/built-in-ruby-methods-jigsaw.md

```diff
@@ -16,7 +16,7 @@ You will be practicing and teaching each other **how to read through the Ruby do
 
 ### Where to Start
 
-You will want to start with the official Ruby docs for your given method. To find the right web page, you may want to directly use a search engine, or you can look through the Ruby docs for [String](https://ruby-doc.org/core-2.5.1/String.html), [Array](https://ruby-doc.org/core-2.5.1/Array.html), and [Hash](https://ruby-doc.org/core-2.5.1/Hash.html). You will want to answer the questions:
+You will want to start with the official Ruby docs for your given method. To find the right web page, you may want to directly use a search engine, or you can look through the Ruby docs for [String](https://ruby-doc.org/core-2.5.5/String.html), [Array](https://ruby-doc.org/core-2.5.5/Array.html), and [Hash](https://ruby-doc.org/core-2.5.5/Hash.html). You will want to answer the questions:
```

Comment:
Oof, not a big fan of the version numbers in these URLs. If you remove the `-2.5.5`, that will link to the most recent version (2.6.x), which is maybe better than having to change these links every time?

Reply:
Another good option would be to write a little `sed` script that can trove through the whole curriculum and change the ruby version in all these links automatically. Then we can just run that once per cohort with whatever version we move to.

Reply:
Ran my bash script to update it to just link to core instead of a specific version number.